fbw-a380x/src/wasm/systems/a380_systems/src/hydraulic/mod.rs

```diff
@@ -474,13 +471,13 @@ impl A380AileronFactory {
         let assembly_middle = Self::a380_aileron_assembly(
             context,
             init_drooped_down,
-            Some(Self::MIDDLE_PANEL_EHA_BUS),
+            Some(AC_EHA_BUS),
```

Comment:
The middle aileron is still powered by the AC ESS bus (427XP to be exact, but that is a part of AC ESS)